python/singa/autograd.py

```diff
@@ -586,6 +586,52 @@ def device_check(self, *inputs):
             if var.device.id() != x_device:
                 var.to_device(x_device)
 
+    def find_sublayers(self):
+        # return a list whose elements are in form of (attribute_name,
+        # sublayer)
```

Comment:
you need to 
1. remove these code from #422 
2. rebase your branch onto the latest master branch. then you will have these code in your branch